src/components/DonatePage/DonatePageBody.tsx

```diff
@@ -288,9 +288,7 @@ const DonatePageBody = () => {
             <div className='border rounded-lg flex relative '>
               <div className='md:w-40 flex gap-4 p-4 border '>
                 <IconMatic size={24} />
-                <h1 className=' font-medium'>
-                  {tokenDetails?.symbol || 'POL'}
-                </h1>
+                <h1 className=' font-medium'>POL</h1>
```

Comment:
we should not use token symbol from configs?

Reply:
Ben asked me to change it to POL because in produtions its gonna be POL only